test/reentrancy/ReentrancyTest.t.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import { Test } from "forge-std/Test.sol";
+import { Deploy } from "../../script/Deploy.s.sol";
+import { DeploymentConfig } from "../../script/DeploymentConfig.s.sol";
+import { AttackController } from "./AttackController.sol";
+import { AttackAccount } from "./AttackAccount.sol";
+import { MiniMeToken } from "../../contracts/MiniMeToken.sol";
+import "forge-std/console.sol";
```

Comment:
Linter already complained about this. Generally, if you need `console` you get it from `forge-std/Test.sol`

Reply:
I didnt find other way.